docs: update Typescript minimum version: v7 + link (#1812)
slackapi/node-slack-sdk: docs/_main/typescript.md

### Minimum version

> Latest major versions of `@slack/web-api`, `@slack/rtm-api`, and `@slack/webhook` packages are supported to build against TypeScript version 5.3.x. You can try to use a greater minor version of Typescript like 5.4 or above, but beware that [API Breaking Changes](https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes) can be introduced i n minor Typescript versions that break compatibility.

Reviewer: ```suggestion
Latest major versions of `@slack/web-api`, `@slack/rtm-api`, and `@slack/webhook` packages are supported to build against TypeScript version 5.3.x. You can try to use a greater minor version of Typescript like 5.4 or above, but beware that [API Breaking Changes](https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes) can be introduced in minor Typescript versions that break compatibility.
```

Author: Whoopsie! Good catch 🦅 👁️ 

Done in af4499b53ec17e0430ca832809166d38c77492d8
